src/content/docs/tutorial-customer-experience/bottom-funnel-analysis.mdx

```diff
@@ -0,0 +1,190 @@
+
+<CollapserGroup>
+    <Collapser
+        id="conversion-rate"
+        title="Bottom-of-the-funnel success/conversion rate"
+    >
+The rate of conversion once a user has gone far enough to demonstrate intent to complete an action to actual completion. Examples of this are rates of:
+* Checkout -> order submission
+* Insurance form review -> submission
+* Completing sign up details -> submission
+
+Increase the rate of conversion by addressing errors and latency at the bottom of the funnel.
+    </Collapser>
+    <Collapser
+        id="latency-risk"
+        title="Revenue at risk due to latency"
+    >
+        Value of a conversion times the number of pages or interactions in the bottom of the funnel that are slower than the industry threshold. Focus on reducing this value by improving page KPIs.
+    </Collapser>
+    <Collapser
+        id="Error risk"
+        title="Revenue at risk due to errors"
+    >
+        Value of a conversion times the number of backend errors in the bottom of the funnel interactions. Tune this value to make it meaningful by filtering out errors that aren't visible to the end user, then focus on reducing it. Create an alert to notify you if it suddenly trends upward.
+    </Collapser>
+</CollapserGroup>
+
+<Steps>
+    <Step>
+### Identify where the bottom of the funnel starts
+
+The bottom of the funnel is focused on the final steps of a user journey where a user has gone far enough to show intent to complete the journey. Here are some examples:
+
+<CollapserGroup>
+    <Collapser
+        id="ecommerce-journey"
+        title="Ecommerce user journey"
+    >
+<img
+  title="Ecommerce bottom of the funnel"
+  alt="Ecommerce bottom of the funnel"
+  src={cxBotfaEcommerce}
+/>
+
+The user journey is simplified so you can focus on where the bottom of the funnel begins: at checkout. Most users entering the checkout phase plan to purchase something. Reducing errors and latency from this point onward is more likely to improve conversions than focusing on any other part of the funnel.
+    </Collapser>
+    <Collapser
+        id="insurance-journey"
+        title="Car insurance purchase user journey"
+    >
+<img
+  title="Car insurance bottom of the funnel"
+  alt="Car insurance bottom of the funnel"
+  src={cxBotfaCarInsurance}
+/>
+
+In the example above, you have the user's interest in car insurance as they enter information, but you don't know their intent until they see the quote and continue to proceed.
+    </Collapser>
+</CollapserGroup>
+    </Step>
+    <Step>
+### Distinguish between pages and actions
+
+The final steps of a user's journey is likely to be a mix of full page loads and AJAX calls. You'll need to know all pages and AJAX requests for the next step. If you're not sure which requests are running from the page in question, you can run:
+
+```
+SELECT count(*) FROM AjaxRequest WHERE pageUrl like '%FILTER%' FACET groupedRequestUrl SINCE 1 DAY AGO
+```        
+
```

Comment:
This section feels incomplete

Reply:
I've added a little more to it to help it segue into the next step, but yes, it's a pretty simple step to take. I'll think about ways to expand on it.